### PR TITLE
Copy edit of the Orchestrator readme

### PR DESCRIPTION
**Pull request by JonathanFingold**
## Proposed Changes
Edit for style and clarity.

## Testing
Visual review of updated .md content.

### CODE REVIEW

**Comment by JonathanFingold** [comment]
@scheyal This is ready for your review. I included additional commentary in HTML comments, which should be removed before this is merged.

File: Orchestrator/README.md [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+**Entity extraction** is not yet supported.
 
-## Authoring Experience
+## Authoring experience
 
 Orchestrator can be used in different development environments:
 
-* **Code First**: Orchestrator can be integrated into your code project by replacing LUIS for intent recognition such as for skill delegation or dispatching to subsequent language understanding services.  See [Runtime Integration](#runtime-integration) section for more.
-* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchestrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
+- [Bot Framework SDK][24]: Orchestrator can be integrated into your code project by replacing LUIS for intent recognition, such as for skill delegation or dispatching to subsequent language understanding services. See the [SDK integration](#sdk-integration) section for more information. <!--We don't yet document Orchestrator in the SDK docs. Do we need to?-->
```
Comment (scheyal) [inline Orchestrator/README.md:27]: Orchestrator is still PREVIEW. We should plan to doc in MS Docs by GA ~BUILD timeframe. Not now.

File: Orchestrator/README.md [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
+<!-- Coining the term "orchestration" doesn't seem necessary. -->
+
+<!-- Discussing the underlying technology also doesn't seem necessary here. Mention in the technical overview, instead. -->
+
+Orchestrator is an LU solution optimized for conversational AI applications. It is built ground-up to run locally with your bot. See the [technical overview][18] for additional details.
 
 ## Scenarios
-**Dispatch**: Orchestrator is a successor to [dispatch][5]. You can use Orchestrator instead of the current dispatch solution to arbitrate across your [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
+
+**Routing**: For bots, Orchestrator can replace the [LUIS Dispatch tool][5]. You can use Orchestrator instead of Dispatch to aggregate multiple [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
```
Comment (scheyal) [inline Orchestrator/README.md:13]: They used the term arbitrate intentionally. Dependening on your perspective, it essentially selects the "correct" subsequent service to service the request. It doesn't really combine, rather selects a winner to route to.

File: Orchestrator/README.md [hunk 3]
```diff
@@ -39,26 +54,32 @@ Thus, use of [BF command line tool][7] to prepare and optimize the model for you
    * See `bf orchestrator:basemodel:list` for alternate models. You may need to experiment with the different models to find which performs best for your language domain.
 3. Combine the label file .lu from (1) with the base model from (2) to create a *snapshot* file with a .blu extension.
    * Use [`bf orchestrator:create`][16] to create just a single .blu snapshot file for all Lu/json/qna tsv files for dispatch scenario.
+-->
+
+<!--This is insufficient information, but again the other article covers this.
```
Comment (scheyal) [inline Orchestrator/README.md:59]: I agree, now that we have the other doc, we can trim this section, and refer to the other doc.

File: Orchestrator/README.md [hunk 4]
```diff
@@ -179,13 +156,13 @@ See the [School skill navigator](https://github.com/microsoft/BotBuilder-Samples
 [12]:https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator/csharp_dotnetcore
 [13]:https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator/javascript_nodejs
 [14]:./docs/API_reference.md
```
Comment (scheyal) [inline Orchestrator/README.md:158]: Change  API reference to: https://aka.ms/bforchestratorapi
(just added that)

File: Orchestrator/README.md (hunk 3 above)
Comment (scheyal) [reply Orchestrator/README.md:59]: I'll leave it up to you to decide what's best here :)

**Review by scheyal — APPROVED** [review]
See comments.

File: Orchestrator/README.md (hunk 3 above)
Comment (JonathanFingold) [reply Orchestrator/README.md]: Yes, please add a prerequisites note to the other article. (I'm keeping a similar note in this article.) I'll call this good for now.

File: Orchestrator/README.md (hunk 2 above)
Comment (JonathanFingold) [reply Orchestrator/README.md]: fixed

File: Orchestrator/README.md (hunk 4 above)
Comment (JonathanFingold) [reply Orchestrator/README.md]: done

**Review by scheyal — COMMENTED** [review]
Good cleanup :)

**Pull request merged**